### PR TITLE
KHWE-56: Implement basic Auth component

### PR DESCRIPTION
**Pull request by lubovgribiniyk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "description": "Simple music streaming service",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.5.1",
+    "@material-ui/styles": "^3.0.0-alpha.0",
```
Comment (gamesminer) [inline package.json]: Please remove "^"

File: src/components/Auth/Auth.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
```
Comment (gamesminer) [inline src/components/Auth/Auth.js]: Group material-ui/core/... in one string. 
import { Tab, Tabs} from "@material-ui/core/Tab";

File: src/components/Auth/Auth.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    maxWidth: "18rem",
+    margin: "0 auto"
+  },
+  tab: {
+    minWidth: "50%"
+  }
+};
+
+class Auth extends Component {
+  state = {
```
Comment (gamesminer) [inline src/components/Auth/Auth.js]: Use PropTypes like this: https://github.com/ihor4568/spotty-react/blob/master/src/components/shared/Main.js

static propTypes = {
  classes: PropTypes.object.isRequired
};

File: src/components/Auth/SignIn.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import Input from "@material-ui/core/Input";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import Button from "@material-ui/core/Button";
+import { withStyles } from "@material-ui/styles";
```
Comment (gamesminer) [inline src/components/Auth/SignIn.js]: Group them

File: src/components/Auth/SignUp.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+import Button from "@material-ui/core/Button";
+import { withStyles } from "@material-ui/styles";
+
+const styles = {
+  button: {
+    marginTop: 30
+  }
+};
+
+class SignUp extends Component {
```
Comment (gamesminer) [inline src/components/Auth/SignUp.js]: Use PropTypes with form, email, name and etc with types

File: src/components/Auth/SignIn.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+    return (
+      <form>
+        <FormControl margin="normal" required fullWidth>
+          <InputLabel htmlFor="email">Email</InputLabel>
+          <Input
+            value={this.state.form.email}
+            onChange={this.handleInputChange}
+            name="email"
+            id="email"
```
Comment (gamesminer) [inline src/components/Auth/SignIn.js]: Do not use id

File: src/components/Auth/SignIn.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+          />
+        </FormControl>
+        <FormControl margin="normal" required fullWidth>
+          <InputLabel htmlFor="password">Password</InputLabel>
+          <Input
+            value={this.state.form.password}
+            onChange={this.handleInputChange}
+            name="password"
+            type="password"
+            id="password"
```
Comment (gamesminer) [inline src/components/Auth/SignIn.js]: Do not use id

Comment (lubovgribiniyk) [reply src/components/Auth/SignIn.js]: I use ids for labels.

File: src/components/Auth/SignIn.js (hunk 6 above)
Comment (lubovgribiniyk) [reply src/components/Auth/SignIn.js]: I use ids for labels.

File: src/components/Auth/SignUp.js (hunk 5 above)
Comment (lubovgribiniyk) [reply src/components/Auth/SignUp.js]: I fixed it only for classes. I do not have other properties in this component.

File: src/components/Auth/SignIn.js (hunk 4 above)
Comment (lubovgribiniyk) [reply src/components/Auth/SignIn.js]: fixed

File: src/components/Auth/Auth.js (hunk 3 above)
Comment (lubovgribiniyk) [reply src/components/Auth/Auth.js]: fixed

File: src/components/Auth/Auth.js (hunk 2 above)
Comment (lubovgribiniyk) [reply src/components/Auth/Auth.js]: fixed

**Pull request merged**